nixos/modules/services/databases/postgresql.nix

```diff
@@ -225,16 +225,6 @@ in
           Contents of the <filename>recovery.conf</filename> file.
         '';
       };
-      superUser = mkOption {
```

Comment:
Should be referenced in `mkRemovedOptionModule`. Also, please keep in mind that there are several other upstream modules relying on this option's value.

Reply:
Wow... I didn't realize the `superUser` was ~~abused~~ used like that :worried: I'm not sure I want to dig into that. I've left the option in but put a `readOnly` value of `postgres` now. Thanks for pointing that out @Ma27 ... you saved the day by mentioning that! :tada: